doc/installation.rst

```diff
@@ -85,57 +94,73 @@ On macOS, the first time you open Spyder, you may see a message that it cannot b
-
 
 
 .. _anaconda_install_ref:
+.. _installation-conda:
 
-========
-Anaconda
-========
+==================
+Anaconda/Miniconda
+==================
 
-Spyder is included by default in the `Anaconda Python distribution`_, which comes with everything you need to get started in an all-in-one package, and is our recommended installation method on Linux (and supported on the other platforms too).
+Spyder is included by default in the `Anaconda`_ Python distribution, which comes with everything you need to get started in an all-in-one package, and can easily be installed in the much lighter-weight `Miniconda`_, which includes just Python and the Conda package and environment manager by default.
+This is our recommended installation method on Linux and for users needing third-party Spyder plugins, and supported on Windows and macOS too.
 
-.. _Anaconda Python distribution: https://www.anaconda.com/products/distribution
+.. _Anaconda: https://www.anaconda.com/products/distribution
+.. _Miniconda: https://conda.io/miniconda.html
 
 
+.. _installation-anaconda-running:
+
 Running with Anaconda
 ~~~~~~~~~~~~~~~~~~~~~
 
-To run Spyder after installing it with Anaconda, the recommended method on Windows is to launch it via the Start menu shortcut.
+To run the bundled version of Spyder after installing it with Anaconda, the recommended method on Windows is to launch it via the Start menu shortcut.
 On other platforms, open Anaconda Navigator, scroll to Spyder under ``Home``, and click ``Launch``.
 
 .. image:: /images/installation/installation-anaconda-navigator.png
    :alt: Anaconda Navigator showing Spyder
 
 If Spyder does not launch via this method or you prefer to use the command line, open Anaconda Prompt (Windows) or your terminal (other platforms), type ``conda activate base`` then ``spyder``.
 
+.. note::
 
-New Conda environment
-~~~~~~~~~~~~~~~~~~~~~
+   The bundled Spyder version can often be quite out of date, missing new features and bug fixes from the latest version, and if you install, change or remove other packages, there is a chance of dependency conflicts or a broken Spyder installation.
+   We recommend :ref:`installing Spyder into a new Conda environment <installation-miniconda>` to avoid all these issues.
 
-If you would like to have Spyder in a dedicated environment to update it separately from your other packages and avoid any conflicts, you can.
-Just run the following command in your Anaconda Prompt (Windows) or terminal (other platforms), for a minimal install of Spyder into a new environment called ``spyder-env``:
 
-.. code-block:: bash
+.. _installation-miniconda:
 
-   conda create -n spyder-env spyder
+Installing with Miniconda
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To install Spyder's optional dependencies as well for full functionality, use the following command instead:
+Once you download and install Miniconda, or to get a more reliable and up-to-date Spyder install with Anaconda, install Spyder into its own fresh environment.
```

Comment:
```suggestion
Once you download and install Miniconda, or to get a more reliable and up-to-date Spyder version with Anaconda, we recommend installing Spyder into its own fresh environment.
```

* Avoid repeating `install` three times in this sentence.
* Mention explicitly that we recommend this.

Reply:
> Avoid repeating install three times in this sentence.

Yeah, I actually already rewrote this sentence to avoid that.

> Mention explicitly that we recommend this.

Reworded a bit, but I implemented the intent here